src/js/core/calculateLottoPrize.js

```diff
@@ -0,0 +1,94 @@
+import { LOTTO_INFO, LOTTO_PRIZE_INFO } from '../constants/constant.js';
+
+export default class CalculateLottoPrize {
+  constructor(currentLottoNumbers, lastLottoNumberList) {
+    this.currentLottoNumbers = currentLottoNumbers;
+    this.lastLottoNumbers = lastLottoNumberList.slice(
+      0,
+      lastLottoNumberList.length - 1,
+    );
+    this.bonusNumber = lastLottoNumberList[lastLottoNumberList.length - 1];
+    this.totalPrizeMoney = 0;
+    this.lottoRankObj = {
+      rank_5: 0,
+      rank_4: 0,
+      rank_3: 0,
+      rank_2: 0,
+      rank_1: 0,
+    };
+  }
+
+  getLottoPrize = (lotto, correctCount) => {
+    let prizeMoney = 0;
+    if (
+      correctCount === LOTTO_PRIZE_INFO.MATCH_NUMBER_5 &&
+      lotto.numbers.includes(this.bonusNumber)
+    ) {
+      return LOTTO_PRIZE_INFO.MATCH_NUMBER_BONUS_PRIZE;
+    }
+    switch (correctCount) {
+      case LOTTO_PRIZE_INFO.MATCH_NUMBER_3:
+        prizeMoney = LOTTO_PRIZE_INFO.MATCH_NUMBER_3_PRIZE;
+        break;
+      case LOTTO_PRIZE_INFO.MATCH_NUMBER_4:
+        prizeMoney = LOTTO_PRIZE_INFO.MATCH_NUMBER_4_PRIZE;
+        break;
+      case LOTTO_PRIZE_INFO.MATCH_NUMBER_5:
+        prizeMoney = LOTTO_PRIZE_INFO.MATCH_NUMBER_5_PRIZE;
+        break;
+      case LOTTO_PRIZE_INFO.MATCH_NUMBER_6:
+        prizeMoney = LOTTO_PRIZE_INFO.MATCH_NUMBER_6_PRIZE;
```

Comment:
https://github.com/woowacourse/javascript-lotto/pull/123/files#diff-b58722b0026e866c0188afac6e83021747d56cb438e632cb34d89e5f99e13304R27 에서 바로 리턴한 것처럼 여기서도 바로 리턴해주면 어떨까요?
```suggestion
        return LOTTO_PRIZE_INFO.MATCH_NUMBER_6_PRIZE;
```